.prettierignore

```diff
@@ -1,10 +1,4 @@
-/.docusaurus
-/.vscode
-/.yarn
-/build
-/node_modules
```

Comment:
Not sure why we are including node_modules?

Reply:
Prettier automatically looks inside `.gitignore`, so it's redundant to ignore it there as well